charts/tuf/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: tuf
+description: A framework for securing software update systems - the scaffolding implementation
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
What does `1.16.0` refer to?

Reply:
Good catch! When I wrote this number, I know had some logic behind it.
Unfortunately, I can't quite get it back together.
I changed the number to match the version of the tuf image that is used here:
https://github.com/sigstore/scaffolding/pkgs/container/scaffolding%2Fserver/38864075?tag=v0.4.6
0a2b3df